### PR TITLE
doctl 1.1.0

### PR DESCRIPTION
**Pull request by aybabtme**
updated version

### CODE REVIEW

File: Formula/doctl.rb [hunk 1]
```diff
@@ -26,9 +26,9 @@ def install
     doctl_version = version.to_s.split(/\./)
     base_flag = "-X github.com/digitalocean/doctl"
     ldflags = %W[
-      #{base_flag}.Major=#{doctl_version[0]}
+      #{base_flag}.Major=#{doctl_version[1]}
       #{base_flag}.Minor=#{doctl_version[1]}
```
Comment (mikemcquaid) [inline Formula/doctl.rb]: You're using `1` twice, here. Is the numbering reversed now or something?

**Comment by mikemcquaid** [comment]
The test is broken; looks like the reported version number is incorrect: http://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/827/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/test_doctl/

File: Formula/doctl.rb (hunk 1 above)
Comment (aybabtme) [inline Formula/doctl.rb]: I just misunderstood what this array was 😢

File: Formula/doctl.rb [hunk 2]
```diff
@@ -3,8 +3,8 @@
 class Doctl < Formula
   desc "Command-line tool for DigitalOcean"
   homepage "https://github.com/digitalocean/doctl"
-  url "https://github.com/digitalocean/doctl/archive/v1.0.2.tar.gz"
-  sha256 "7a28bd0f9e68eee41761e32267a0d63329c6f0e16e0d8b5afb4a8286b914e98a"
+  url "https://github.com/digitalocean/doctl/archive/v1.1.0.tar.gz"
```
Comment (mikemcquaid) [inline Formula/doctl.rb]: This link is now broken: http://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/887/version=yosemite/testReport/junit/brew-test-bot/yosemite/install_doctl/

Comment (mikemcquaid) [inline Formula/doctl.rb]: https://github.com/digitalocean/doctl/releases/tag/v1.0.2 is the latest published tag, it seems.

Comment (aybabtme) [inline Formula/doctl.rb]: Yup, sorry about that. I'll reopen when that's solved to avoid wasting more of you time. Apologies!

**Pull request closed without merging**